lk2nd/device/dts/msm8916/msm8939-huawei-rio.dts

```diff
@@ -12,8 +12,22 @@
 };
 
 &lk2nd {
-	model = "Huawei G7 Plus / G8 / GX8";
+	model = "Huawei G7 Plus / G8 / GX8 / Maimang 4";
```

Comment:
> Maimang 4

Would be better if it's written in the commit message, or split into another commit.

Reply:
Other than that, looks good to me.